### PR TITLE
[SDESK-629] Adding Desk/Stage information in item preview view

### PR DESCRIPTION
**Pull request by nrvikas**

### CODE REVIEW

File: scripts/apps/search/directives/ItemPreview.js [hunk 1]
```diff
@@ -45,6 +45,14 @@ export function ItemPreview(asset, storage) {
 
             scope.$watch('item', (item) => {
                 scope.selected = {preview: item || null};
+
+                // Set the desk and stage names
+                if (item && item.task && item.task.stage) {
+                    scope.deskName = desks.deskLookup[item.task.desk].name;
+                    scope.stage = desks.stageLookup[item.task.stage];
```
Comment (akintolga) [inline scripts/apps/search/directives/ItemPreview.js]: A minor point: if you could do `scope.stage = desks.stageLookup[item.task.stage].name` here then you need only one `<span>` in preview.html (the 2nd one).

Comment (nrvikas) [inline scripts/apps/search/directives/ItemPreview.js]: Absolutely. Will do.

**Comment by petrjasek** [comment]
@nrvikas are you going to update it?

**Comment by petrjasek** [comment]
thx

**Comment by nrvikas** [comment]
@petrjasek . Apologies for letting this slip. Thanks for your patience. Good day :)

**Pull request merged**